setup.py

```diff
@@ -60,7 +62,7 @@ def read(*rnames):
       zip_safe=False,
       install_requires=[
           'setuptools',
-          'z3c.pt >= 2.1',
+          'z3c.pt >= 3.0.0a1',
```

Comment:
Otherwise the tests will fail under Python 3 (see [Travis](https://travis-ci.org/zopefoundation/z3c.ptcompat/jobs/40151146)) because tox only uses non-final versions if it is forced to.